web_ui/packages/smart-tools/src/grabcut/grabcut.ts

```diff
@@ -307,10 +309,25 @@ class Grabcut {
     }
 
     cleanModels(): void {
-        this.img?.delete();
-        this.mask?.delete();
-        this.bgdModel?.delete();
-        this.fgdModel?.delete();
+        if (this.img) {
```

Comment:
not only do we need to delete the mask but also to get rid of the reference (i was getting an error "Mask is already deleted!") so it seems that this more verbose approach is safer

Reply:
This happens now because i added a "loadImage" method here on the grabcut class. Which cleans the models always before creating new masks